api/src/cli/index.test.ts

```diff
@@ -1,25 +1,30 @@
 import { Command } from 'commander';
 import { Extension, HookConfig } from '@directus/shared/types';
 import { createCli } from './index';
+import { test, expect, vi, beforeEach } from 'vitest';
 
-jest.mock('../../src/env', () => ({
-	...jest.requireActual('../../src/env').default,
-	EXTENSIONS_PATH: '',
-	SERVE_APP: false,
-	DB_CLIENT: 'pg',
-	DB_HOST: 'localhost',
-	DB_PORT: 5432,
-	DB_DATABASE: 'directus',
-	DB_USER: 'postgres',
-	DB_PASSWORD: 'psql1234',
-}));
+vi.mock('../../src/env', async () => {
+	const actual = await vi.importActual('../../src/env');
+
+	return {
+		EXTENSIONS_PATH: '',
+		SERVE_APP: false,
+		DB_CLIENT: 'pg',
+		DB_HOST: 'localhost',
+		DB_PORT: 5432,
+		DB_DATABASE: 'directus',
+		DB_USER: 'postgres',
+		DB_PASSWORD: 'psql1234',
+		...(actual as object),
```

Comment:
Not sure if we're overriding the values, but let's keep it the same as before:

```suggestion
		...(actual as object),
		EXTENSIONS_PATH: '',
		SERVE_APP: false,
		DB_CLIENT: 'pg',
		DB_HOST: 'localhost',
		DB_PORT: 5432,
		DB_DATABASE: 'directus',
		DB_USER: 'postgres',
		DB_PASSWORD: 'psql1234',
```